regex-automata/src/nfa/thompson/nfa.rs

```diff
@@ -1260,6 +1266,7 @@ pub(super) struct Inner {
     /// zero-length prefix for any of the patterns in this NFA.
     look_set_prefix_all: LookSet,
     */
+    look_count: usize,
```

Comment:
I would name it `lookaround_count` to avoid confusion with the assertion looks

Reply:
good point